src/Model/helpers.js

```diff
@@ -21,3 +21,24 @@ export const createTimestampsFor = (model: Model) => {
 
   return timestamps
 }
+
+export async function fetchChildren(model: Model): Promise<Model[]> {
+  const { associations } = model.collection.modelClass
+  const childrenKeys = Object.keys(associations).filter(key => associations[key].type === 'has_many')
+  
+  const promises = childrenKeys.map(async key => {
+    const children = await model[key].fetch()
+    const childrenPromises = children.map(async child => {
+      return fetchChildren(child)
+    })
+    const grandchildren = await Promise.all(childrenPromises)
+    let result = []
+    grandchildren.forEach(elt => {result = [...result, ...elt]})
```

Comment:
do not use local variable mutation -- use `map`. also `elt` is a Pythonism, say `item` or `record` or `grandchild`